libraries/mpxe-store/src/store.c

```diff
@@ -50,8 +65,24 @@ Store *prv_get_first_empty() {
 
 static void prv_handle_store_update(uint8_t *buf, int64_t len) {
   MxStoreUpdate *update = mx_store_update__unpack(NULL, (size_t)len, buf);
-  s_func_table[update->type].update_store(update->msg, update->mask, (void *)update->key);
-  mx_store_update__free_unpacked(update, NULL);
+  if (update->type == MX_STORE_TYPE__CMD) {  // Abstract out into another s_func_table func?
+    MxCmd *cmd = mx_cmd__unpack(NULL, (size_t)update->msg.len, update->msg.data);
+    if (cmd->cmd == MX_CMD_TYPE__FINISH_INIT_CONDS) {
+      s_cmd_cb_lookup[MX_CMD_TYPE__FINISH_INIT_CONDS](NULL);
+    }
```

Comment:
A few comments on this block:
1. You need to free anything you unpack, so at the end call `mx_cmd__free_unpacked(cmd, NULL)` (probably not necessary to set it to `NULL` since it goes out of scope)
2. In the second `if`, it's cleaner to do something like:
```
if (s_cmd_cb_lookup[cmd->cmd] != NULL) {
  s_cmd_cb_lookup[cmd->cmd](NULL);
}
```
Because this way it'll work for all command types rather than having a new `if` for each one. I'm not convinced we'll need context for these, but I'm ok leaving it there for now as just always `NULL`. In theory if we want to add more metadata to commands we can just pass the command instead of a `void *`, but we can worry about that later.

3. To apply comment 2 safely, you'll also need to add an `END` command type, similar to store types. This is so you can set `[END]` to `NULL` in `s_cmd_cb_lookup` to properly size the array, otherwise you could end up doing an out-of-bounds access. Also if you do this, you don't have to set all command callbacks to `NULL` because they'll default to `0` by being statically initialized.